package.json

```diff
@@ -10,10 +10,7 @@
     "sharkdown": "sharkdown"
   },
   "dependencies": {
-    "expect.js": "~0.2.0",
```

Comment:
These shouldn't be dependencies, but they need to be devDependencies - the tests rely on them.

Reply:
Expect and stream-spigot already are, but I couldn't find any references to through

Reply:
Okay, yeah - so through should be removed.